src/sentry/grouping/enhancement-configs/common@2019-03-23.txt

```diff
@@ -51,14 +51,15 @@ family:native package:**/Frameworks/KSCrash.framework/**             -app
 family:native package:**/Frameworks/SentrySwift.framework/**         -app
 family:native package:**/Frameworks/Sentry.framework/**              -app
 
-# Sentry internal functions in Cocoa SDK
+# Needed for versions < sentry-cocoa 6.3.0 and static linking.
+# Before sentry-cocoa 6.3.0, we marked all packages located inside the application bundle as inApp.
+# Since 6.3.0, the Cocoa SKD only marks the main executable as inApp. This change doesn't impact
+# applications using static libraries, as when using static libraries, all of them end up in the
+# main executable.
 family:native function:kscm_*                                     -app -group
 family:native function:sentrycrashcm_*                            -app -group
 family:native function:kscrash_*                                  -app -group
 family:native function:sentrycrash_*                              -app -group
-family:native function:"?[[]KSCrash *"                            -app -group
-family:native function:"?[[]SentryCrash *"                        -app -group
-family:native function:"?[[]SentryClient *"                       -app -group
-family:native function:"?[[]SentryHub *"                          -app -group
-family:native function:"?[[]RNSentry *"                           -app -group
-family:native function:"?[[]SentrySDK *"                          -app -group
+family:native function:"?[[]KSCrash*"                             -app -group
+family:native function:"?[[]RNSentry*"                            -app -group
+family:native function:"?[[]Sentry*"                              -app -group
```

Comment:
Downside of this is that it might affect test apps that folks create to test the SDK.
Often call it "SentryTest" for example, and some confusion arises. That could be one reason to stick to the original `SentrySDK`, `SentryHub`..

Reply:
That's true. Maybe I just add all classes that could end up in your stacktrace.

Reply:
> Maybe I just add all classes that could end up in your stacktrace.

In the past, when we renamed from KSCrash to SentryCrash (or was it something else?), we forgot to update the list here. This is extremely fragile. Is there no one pattern that we can add instead? This now just creates a maintenance burden.

Reply:
I rolled back to `Sentry*` and added a comment.

Reply:
Thanks. Static linking is unfortunate here, otherwise we could further narrow this down to package :(